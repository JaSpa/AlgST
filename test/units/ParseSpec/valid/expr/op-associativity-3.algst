finally <|
  a == b 
    |> pipe
    |> here
    |> first
