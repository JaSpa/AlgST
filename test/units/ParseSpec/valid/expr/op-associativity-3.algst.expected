Exp.App
|
+- Exp.Var finally
|
`- Exp.App
   |
   +- Exp.Var first
   |
   `- Exp.App
      |
      +- Exp.Var here
      |
      `- Exp.App
         |
         +- Exp.Var pipe
         |
         `- Exp.App
            |
            +- Exp.App
            |  |
            |  +- Exp.Var (==)
            |  |
            |  `- Exp.Var a
            |
            `- Exp.Var b

